packages/adapter-utils/src/collisions.ts

```diff
@@ -106,6 +109,14 @@ export const getCollisionWarnings = ({
     _.groupBy(instances, instance => instance.elemID.getFullName()),
     collideInstances => collideInstances.length > 1,
   )
+
+  Object.entries(elemIDtoInstances).forEach(([elemID, collideInstances]) => {
+    const relevantInstanceValues = collideInstances.map(instance => _.pickBy(instance.value, val => val != null))
+    const relevantInstanceValuesStr = relevantInstanceValues.map(instValues => inspectValue(instValues)).join('\n')
+    log.debug(`Omitted instances with colliding ElemID ${elemID} with values - 
+${relevantInstanceValuesStr}`)
```

Comment:
nit-  string formatting is preferred:
```suggestion
    log.debug('Omitted instances with colliding ElemID %s with values - \n%s', elemID, relevantInstanceValuesStr)
```